content/applications/hr/payroll.rst

```diff
@@ -457,22 +709,101 @@ advantage listed for a particular structure type is only available for that spec
    have meal vouchers. The opposite is true for someone hired under the :guilabel:`Intern` structure
    type. They would have meal vouchers available to them, not the use of a company car.
 
-To make a new advantage, click the :guilabel:`New` button, and enter the information in the fields.
-The required fields for an advantage are:
+To make a new benefit, click the :guilabel:`New` button, and enter the information in the fields on
+the blank benefits form.
 
-- :guilabel:`Name`: Enter the name for the advantage.
-- :guilabel:`Advantage Field`: Select from the drop-down menu what field in the payslip this
-  advantage appears under.
-- :guilabel:`Advantage Type`: Select from the drop-down menu what type of advantage the benefit is.
-  Select from :guilabel:`Monthly Benefit in Kind`, :guilabel:`Monthly Advantages in Net`,
-  :guilabel:`Monthly Advantages in Cash`, or :guilabel:`Yearly Advantages in Cash`.
-- :guilabel:`Salary Structure Type`: Select from the drop-down menu which salary structure type this
-  advantage applies to.
-- :guilabel:`Display Type`: Select from the drop-down menu how this advantage is displayed.
+The various fields for creating a benefit are as follows:
 
-.. image:: payroll/new-advantage.png
+General information section
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- :guilabel:`Benefits`: enter the name for the benefit. This field is required.
+- :guilabel:`Benefit Field`: select from the drop-down menu the corresponding type of field in the
+  contract for this benefit.
+- :guilabel:`Cost Field`: select from the drop-down menu the type of cost incurred by the company
+  for this specific benefit. The default options are :guilabel:`Calendar Changed`, :guilabel:`Yearly
+  Cost (Real)`, :guilabel:`Extra Time Off`, :guilabel:`Hourly Wage`, :guilabel:`Part Time`,
+  :guilabel:`Wage`, :guilabel:`Wage with Holidays`, and :guilabel:`Work time rate`. Depending on the
+  localization settings, additional options are available.
+- :guilabel:`Related Type`: select from the drop-down menu what type of benefit it is. Select from
+  :guilabel:`Monthly Benefit in Kind`, :guilabel:`Monthly Benefit in Net`, :guilabel:`Monthly
+  Benefit in Cash`, :guilabel:`Yearly Benefits in Cash`, or :guilabel:`Non Financial Benefits`. This
+  field is required.
+- :guilabel:`Benefit Field`: select from the drop-down menu the field on the contract the benefit
+  appears.
+- :guilabel:`Cost Field`: select the field on the contract that the cost of the benefit is linked
+  to, using the drop-down menu. If this field is left blank, the cost of the benefit is not computed
+  in the employee budget.
```

Comment:
is this also a duplicate mention?

Reply:
These are similar but different- the first mentions are a reference, while the second instance of these fields are the specific placeholders on the contract. it is REALLY odd and it should NOT appear, but it does. I have a ticket in for this currently, but since they show up, I included them.